doc/user_manual/existing_interfaces.tex

```diff
@@ -4741,4 +4741,101 @@ \subsubsection{Additional Information}
 Neutrino input file, the Neutrino interface must also be changed. Similarly, the Neutrino interface searches for the
 output in the default Measurement field name: '\texttt{MeasurementField\_1}'. Again, this would need to modified in the interface
-if the measurement field name was changed.
\ No newline at end of file
+if the measurement field name was changed.
+
+\subsection{Prescient Interface}
+\label{subsec:prescientInterface}
+
+\subsubsection{General Information}
+The Prescient Interface is used to run the open source Prescient
+production cost modeling platform available from
+\url{https://github.com/grid-parity-exchange/Prescient}
+
+This allows inputs to be perturbed and data to be read out.
+
+For perturbing inputs, there are two formats allowed.  The first is a
+sampled variable can be placed between \$ like \verb'$var$' The second
+requires parentheses and allows the sampled variable to have a
+constant added or a multiplication factor like \verb'$(var+3.2)$' or
+\verb'$(var*2.1)$' or \verb'$(var*5.0+7.0)$' or \verb'$(a*-2.0)$'
+These can be placed in any of the .dat or .csv files that are listed
+in the \xmlNode{Files} section as \xmlAttr{type="PrescientInput"}
+
+
+\subsubsection{Files}
+
+There are two types of inputs in the \xmlNode{Files} section.  The
+\xmlAttr{type="PrescientRunnerInput"} ones are passed as an argument
+to the \texttt{runner.py} If multiple PrescientRunnerInput files are
+specified then \texttt{runner.py} will be called multiple times (which
+can be used to run a populate and then simulate command).  The
+\xmlAttr{type="PrescientInput"} are just used as additional inputs
+that have the data in them perturbed.
+
+\begin{lstlisting}[style=XML]
+  <Files>
+  <Input name="simulate" type="PrescientRunnerInput"
+  >simulate_day.txt</Input>
+    <Input name="structure" type="PrescientInput"
+    subDirectory="scenarios/pyspdir_twostage/2020-07-10/"
+    >ScenarioStructure.dat</Input>
+    <Input name="scenario_1" type="PrescientInput"
+    subDirectory="scenarios/pyspdir_twostage/2020-07-10/"
+    >Scenario_1.dat</Input>
+    <Input name="actuals" type="PrescientInput"
+    subDirectory="scenarios/pyspdir_twostage/2020-07-10/"
+    >Scenario_actuals.dat</Input>
+    <Input name="forcasts" type="PrescientInput"
+    subDirectory="scenarios/pyspdir_twostage/2020-07-10/"
+    >Scenario_forecasts.dat</Input>
+    <Input name="scenarios" type="PrescientInput"
+    subDirectory="scenarios/pyspdir_twostage/2020-07-10/"
+    >scenarios.csv</Input>
+  </Files>
+\end{lstlisting}
+
+\subsubsection{Models}
+
+The \xmlNode{Code} model can be used with the
+\xmlAttr{subType="Prescient"} to run the Prescient Code Interface.
+
+\begin{lstlisting}[style=XML]
+  <Models>
+    <Code name="TestPrescient" subType="Prescient">
+      <executable>
+      </executable>
+    </Code>
+  </Models>
+\end{lstlisting}
+
+\subsubsection{Installation of Libraries}
```

Comment:
Josh, a section abut the specification of the "Code" block and the type of outputs that are extractable should be placed in the manual.
I would add, at least, a section  **Models**, a section  **Output Files Conversion** , a section **Sampler** (to show an example on how the variables should be inputted).  Take a look at the other code interfaces as an example (e.g. PHISICS, CTF, etc)

Reply:
Sections added.

Reply:
Josh, we need to be a bit more verbose here. Please add examples of the data variable names in the Output Files Conversion. The format of the output (data-hours), the variable names examples, etc.

Reply:
I added an example history set and listed typical variables.